pmoves/services/hi-rag-gateway-v2/geometry_bus.py

```
@@ -67,6 +67,26 @@
+        sys.path.insert(0, str(Path(__file__).resolve().parent))
+    from hrm import HRMSidecar, HRMSidecarConfig as _HRMCfg
+    from config import HRM_ENABLED, DEVICE, HRM_FAST_THRESHOLD, HRM_MAX_PONDER_STEPS, HRM_HALT_THRESHOLD
+    _enhanced_hrm = HRMSidecar(_HRMCfg(
+        enabled=HRM_ENABLED,
+        device=DEVICE,
+        fast_threshold=HRM_FAST_THRESHOLD,
+        max_ponder_steps=HRM_MAX_PONDER_STEPS,
+        halt_threshold=HRM_HALT_THRESHOLD,
+    ))
+    logging.getLogger("hirag.gateway.v2").info("Enhanced HRM dual-stream sidecar initialised (enabled=%s)", HRM_ENABLED)
+except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)